src/app.js

```diff
@@ -77,9 +76,11 @@ app.get("/api-docs.json", (req, res) => {
   res.setHeader("Content-Type", "application/json");
   res.send(swaggerSpec);
 });
-app.use("/user", UserRouter);
-app.use("/role", roleRouter);
-app.use("/permission", permissionRouter);
+app.use("/api/v1/user", UserRouter);
+
+app.use("/api/v1/role", roleRouter);
```

Comment:
Kindly follow the naming conventions for resource names. Since we're not using singleton resources, kindly update from singular nouns to plural ones. `/api/v1/role` => `/api/v1/roles`